section4/README.md

```diff
@@ -55,22 +55,22 @@ Create a new Gist. In it, write a letter introducing yourself to your future ins
 
 Make sure to include your full name and pronouns.
 
-- `Add Link to Gist that contains your letter - here`
+[Please see my letter here.](https://gist.github.com/ohClaire/c54fed773348b82eeebebcfce24faf22) 💌
```

Comment:
Fantastic window into why you chose Turing and software development. Love to read about all the self exploration you've done so far! You're going to do great at Turing. Thanks for your effort on this letter and the entire project!